ambry-api/src/main/java/com/github/ambry/config/RouterConfig.java

```diff
@@ -500,6 +503,10 @@
   @Default("0")
   public final int routerBackgroundDeleterMaxConcurrentOperations;
 
+  @Config(ROUTER_PUT_REQUEST_CRC32_IMPLEMENTATION)
+  @Default(CRC32_IMPLEMENTATION_AMBRY_UTIL)
+  public final String routerPutRequestCrc32Implementation;
```

Comment:
You could make this an enum config with `VerifiableProperties.getEnum`. It will avoid having to do a string comparison in `getCrc32Implementation`.

Reply:
i will use a boolean here.